tests/pt-br/light_HassLightSet.yaml

```diff
@@ -1,6 +1,65 @@
 language: pt-br
 tests:
-  - sentences: []
+  # brightness
+  - sentences:
+      - põe o brilho da luz do quarto a 50%
+      - ponha o brilho da luz do quarto a 50%
+      - põe a luz do quarto a 50%
+      - põe a luz do quarto a 50 porcento
+      - definir a luz do quarto a 50 porcento
     intent:
       name: HassLightSet
-      slots: {}
+      slots:
+        brightness: 50
+        name: Luz do Quarto
+    response: Brilho definido
+
+  - sentences:
+      - põe o brilho do quarto a 50%
+      - põe o brilho das luzes do quarto a 50%
+      - põe o brilho das luzes no quarto a 50%
+      - põe o brilho de todas as luzes do quarto a 50%
+      - põe todas as luzes do quarto a 50%
+    intent:
+      name: HassLightSet
+      slots:
+        brightness: 50
+        area: Quarto
+        name: all
+    response: Brilho definido
+
+  # color
+  - sentences:
+      - põe a luz do quarto a vermelho
```

Comment:
Here we can change it to "para vermelho", what do you think?

Reply:
Ok Here, but there is a lot of sentences from pt from Portugal,
I will make another PR later on to correcet these senteces.
Many of them are not really used in pt-br.
If we agree we can keep like that for now and update this later.